### PR TITLE
master-qa-59363

### PR DESCRIPTION
**Pull request by dellawang**
https://issues.liferay.com/browse/LRQA-59363

### CODE REVIEW

**Comment by dellawang** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:bundle

**Comment by gaborlovasdotcom** [comment]
ci:test:security

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/mfa/MultiFactorAuthentication.testcase [hunk 1]
```diff
@@ -572,6 +572,41 @@ definition {
 			userPassword = "test");
 	}
 
+	@description = "This is a use case for LPS-89360"
+	@priority = "4"
+	test ViewOrderOfOTPVerifier {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/mfa/MultiFactorAuthentication.testcase:577]: Could you please add property test.name.skip.portal.instance = "ViewOrderOfOTPVerifier"; ?

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security

**Comment by gaborlovasdotcom** [comment]
Continued in: https://github.com/gaborlovasdotcom/liferay-portal/pull/32

**Comment by gaborlovasdotcom** [comment]
ci:test:security

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**